cmd/handler.go

```diff
@@ -112,6 +112,13 @@ var handlerCmd = &cobra.Command{
 		} else {
 			ctx.Warn("AMQP is not enabled in your configuration")
 		}
+
+		if viper.GetString("handler.device-builtin") != "" {
```

Comment:
Shouldn't the ttn builtin attributes be hardcoded in the binary? We can specify additional attributes with the `--extra-device-attributes` config